test/test_sim.py

```diff
@@ -183,7 +183,7 @@ def test_sim_13(datafiles, simulator):
     with datafiles.as_cwd():
         subprocess.check_call(['runSVUnit', '-s', simulator])
 
-        expect_string(br"ERROR: \[50\]\[dut_ut\]: fail_if: svunit_timeout (at `pwd`/./dut_unit_test.sv line:62)", 'run.log')
+        expect_string(br"ERROR: \[50\]\[dut_ut\]: fail_if: svunit_timeout \(at .*./dut_unit_test.sv line:62\)", 'run.log')
```

Comment:
Strictly speaking, to be congruent to the code before, it should be `.*/./dut_unit_test.sv` (note the extra `/`). I don't think the fact that a `./` is introduced in the path is a worthy detail, so I'll fix this up.